test/e2e/utils.go

```diff
@@ -213,6 +215,15 @@ func undeployJaegerInstance(jaeger *v1.Jaeger) {
 	require.NoError(t, err)
 }
 
+func randomPortNumber() string {
```

Comment:
Random port implementations usually return a random port which is free. There is maybe a syscall which you could use to get the random port.

Note that k8s port-forward provides random port feature which I tried to use in https://github.com/jaegertracing/jaeger-operator/pull/288. To use it we would need to update k8s API to 1.15.

Reply:
@pavolloffay  I've updated this so it should now return a random port which is free.  Can you review and merge if it looks ok?

Reply:
Could you please move this function to `portForward.go` and leave a TODO comment saying use port-forward from k8s once we upgrade k8s API to 1.15.0+